mobile/fastlane/flake.nix

```diff
@@ -0,0 +1,59 @@
+{
+  description = "Fastlane environment for iOS signing";
+
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-25.05";
+    flake-utils.url = "github:numtide/flake-utils";
+  };
+
+  outputs = { self, nixpkgs, flake-utils }:
+    flake-utils.lib.eachDefaultSystem (system:
```

Comment:
I'd rather avoid `flake-utils` input since the same thing can be achieved with a few simple functions:
```nix
      supportedSystems = [
        "x86_64-linux" "i686-linux" "aarch64-linux"
        "x86_64-darwin" "aarch64-darwin"
      ];
      forAllSystems = f: nixpkgs.lib.genAttrs supportedSystems (system: f system);
      pkgsFor = forAllSystems (system: import nixpkgs { inherit system; });
```
https://github.com/logos-messaging/logos-messaging-go/blob/f40bcd7e797881dfccfa57fb70ba58e7dd3cf560/flake.nix#L8-L13